### PR TITLE
PHP lcobucci/jwt: add EdDSA support

### PR DESCRIPTION
**Pull request by Slamdunk**
Waiting for https://github.com/lcobucci/jwt/pull/605 to be merged to get this PR ready-for-review

### CODE REVIEW

File: views/website/libraries/24-PHP.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
         "rs256": true,
         "rs384": true,
         "rs512": true,
+        "eddsa": true,
```
Comment (lcobucci) [inline views/website/libraries/24-PHP.json:87]: This should be on the object above, though :smile:

Comment (Slamdunk) [reply views/website/libraries/24-PHP.json:87]: God, sorry!

**Comment by Slamdunk** [comment]
EdDSA will be availabe from `4.1.0`: should we raise the `minimumVersion` from `3.0.0` @Sambego ?

**Comment by Sambego** [comment]
Yeah, the minimum version should reflect all capabilities set to `true` here :)

**Review by Sambego — APPROVED** [review]

**Pull request merged**